src/domain/session/leftpanel/RoomTileViewModel.js

```diff
@@ -75,6 +76,10 @@ export class RoomTileViewModel extends BaseTileViewModel {
         return timeDiff;
     }
 
+    get avatarColorNumber() {
```

Comment:
You can put this in the parent class, `BaseTileViewModel`, which already has this getter. There, this._avatarSource is either an Invite or a subclass of BaseRoom, so `this._avatarSource.avatarColorId` will handle all cases.